lib/metadata-storage.ts

```diff
@@ -0,0 +1,121 @@
+import { u8aToString } from "@polkadot/util";
+import { u8aConcat } from "@polkadot/util/u8a";
+import { MetadataStorage, createStorage } from "@zeitgeistpm/sdk";
+import { Codec, JsonCodec } from "@zeitgeistpm/utility/dist/codec";
+import * as O from "@zeitgeistpm/utility/dist/option";
+import * as Te from "@zeitgeistpm/utility/dist/taskeither";
+import { Storage, StorageError } from "@zeitgeistpm/web3.storage";
+import * as IPFSHTTPClient from "ipfs-http-client";
+import { CID } from "multiformats/cid";
+
+const node = IPFSHTTPClient.create({ url: "https://ipfs.zeitgeist.pm" });
```

Comment:
Why is that hardcoded?

Reply:
Doesnt need to be.

Reply:
Hmm I think it is sufficient though, don't see why we would want to change this.